### DISCUSSION TITLE
Not Updating to New GHELPER version cause any issues in the future?

### DISCUSSION THREAD

**Discussion by RAGE-HERO7** [seerge/g-helper#1939] [Q&A] [answered]
@seerge is it necessary to keep updating to new ghelper versions if the old build works fine ?

1. will the driver update section in ghelper still works with old build?

2. does upcoming windows updates,system control interface update, bios updates, graphics driver updates causes issues in the future with old GHELPER build ?

3.i will still update though but asking for my non tech savy friends :)

**Comment by seerge** [comment, marked as answer]
It's not necessarily, but then you will miss new features and improvements :)
